docs/domain/domain-model.gql

```diff
@@ -0,0 +1,165 @@
+scalar Hash
+scalar MsFromPosixEpoch
+scalar Base64Url
+scalar Int64
+scalar BigInt
+scalar Base64UrlTransaction
+scalar Object
+
```

Comment:
There is only a well defined fixed set of Chains for a given version of this API. Next time the chainweb graph changes, we would have to update this API in any case.

```suggestion
scalar ChainId

```

Or maybe even use an enum?

```gql
enum ChainId
    CHAIN_0
    CHAIN_1
    CHAIN_2
    # ...
    CHAIN_19
```